package.json

```diff
@@ -22,6 +22,7 @@
         "lint:test": "node_modules/eslint/bin/eslint.js --config=.eslintrc.test.config.js \"test/**/*.spec.js\"",
         "lint": "npm run lint:src && npm run lint:test && npm run docs:lint",
         "precommit": "lint-staged",
+        "prepare": "npm run dist:transpile",
```

Comment:
Although it's more intuitive I'm trying to avoid redundancy in the npm scripts, due to the number of scripts that we have in place already it makes the discovery process a little bit more overwhelming, that's why I would ask you to remove this.

If you like it for development I would advise you to set up an alias in your .bashrc for instance `alias prepare=npm run dist:transpile` and you can customize your shortcuts in a more customized way.

Reply:
I understand the concern for too many scripts, it was a little confusing for me to figure out which scripts were necessary when working locally.  The `prepare` script seems critical to me though as it is one of the  [npm supported scripts](https://docs.npmjs.com/misc/scripts.html) and is run when installing the package directly from GitHub:

```
npm install git+https://git@github.com/danielcaldas/react-d3-graph.git
```

From the [npm docs](https://docs.npmjs.com/cli/install): 

> If the package being installed contains a `prepare` script, its `dependencies` and `devDependencies` will be installed, and the prepare script will be run, before the package is packaged and installed.

I'm relatively new to the JS world, so if there is a more idiomatic or preferred way of installing the package, let me know.

Reply:
I see now. You are installing `react-d3-graph` from the source on git which does not contain the transpired code (this one is only upload to npm). Maybe I could upload the `lib` and put it under the version control, but I need to give a little more thought to this before choosing an approach.

The only problem of `prepare` is that it will generate the transpiled code locally, you can have different versions of babel resulting on different compiled code. By uploading the `lib` to `npm` I'm ensuring that everyone will be consuming the same code, that is why I'll still prefer not to have the `prepare` script, but thanks for bringing this to discussion and also it's good that this is documented here so that more people running into the same issue can solve the problem with ease (hopefully).